ocaml/xapi/helpers.ml

```diff
@@ -772,7 +772,7 @@ let get_my_pbds __context =
 (* Return the PBD for specified SR on a specific host *)
 (* Just say an SR is shared if it has more than one PBD *)
 let is_sr_shared ~__context ~self =
-  List.length (Db.SR.get_PBDs ~__context ~self) > 1
+  match Db.SR.get_PBDs ~__context ~self with [] | [_] -> false | _ -> true
```

Comment:
Isn't this functionally the same as before?

Reply:
indeed, this removes the usage of  list.length